deployment/kubernetes/backstage/helm/values.yaml

```diff
@@ -22,10 +22,11 @@ container:
 secrets:
   backstage: 'backstage-secrets'
   postgres: 'postgres-secrets'
-  ghToken: 'dummy'
-  azToken: 'dummy'
+  jiraTOKEN: 'Basic c2FueWFtc2hhcm1hOEBnbWFpbC5jb206QVRBVFQzeEZmR0YwZWhRWWdVSXBfeXBtbHpOd1lJcVdqUm9vVFRyLXM5Vkpnd3lZTkhDRjFCY0I4Q1dGZXdLNDViTU5jbGtmRldmZkJFSDZtaHI2WFVlbGVka3JrNEhZSDQxYi15bnlRU2lqQm81dF9pOVd2dnlsSllWRHJENmVuU2VoTkk5Tl94X3gySTZiLUdVNEFrXzRRQU5mc2FkRkFBVHdMSER3dFJUV0VCNl9kU0Jzam9NPTJBMTZFREY1'
```

Comment:
Change the token value to a dummy value. Include proper comment on how to encode the Jira API token for usage.